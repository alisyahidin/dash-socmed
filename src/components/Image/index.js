import React from 'react'
import { Link } from 'react-router-dom'
import { Image as BsImage } from 'react-bootstrap'

const Image = props => (
  <Link to="#">
    <BsImage {...props} />
  </Link>
)

export default Image