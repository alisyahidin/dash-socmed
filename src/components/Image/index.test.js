import React from 'react'
import { shallow } from 'enzyme'
import Image from './index'

it('render <Image /> correctly', () => {
  const component = shallow(<Image />)

  expect(component).toMatchSnapshot()
})