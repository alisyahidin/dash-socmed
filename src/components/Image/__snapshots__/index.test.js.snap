// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render <Image /> correctly 1`] = `
<Link
  replace={false}
  to="#"
>
  <ForwardRef(Bootstrap(Image)) />
</Link>
`;
